mobile/test/common/integration/xds_integration_test.cc

```diff
@@ -24,21 +18,14 @@ using ::testing::AssertionResult;
 using ::testing::AssertionSuccess;
 
 XdsIntegrationTest::XdsIntegrationTest() : BaseClientIntegrationTest(ipVersion()) {
-  Grpc::forceRegisterDefaultGoogleGrpcCredentialsFactory();
-  Config::forceRegisterAdsConfigSubscriptionFactory();
-  Config::forceRegisterGrpcConfigSubscriptionFactory();
-  Config::forceRegisterDeltaGrpcConfigSubscriptionFactory();
-  Config::forceRegisterDeltaGrpcCollectionConfigSubscriptionFactory();
-  Config::forceRegisterAggregatedGrpcCollectionConfigSubscriptionFactory();
-  Config::forceRegisterAdsCollectionConfigSubscriptionFactory();
-  Config::forceRegisterGrpcMuxFactory();
-  Config::forceRegisterNewGrpcMuxFactory();
-  Extensions::TransportSockets::Tls::forceRegisterDefaultCertValidatorFactory();
-
   create_xds_upstream_ = true;
   tls_xds_upstream_ = true;
+  skip_tag_extraction_rule_check_ = true;
```

Comment:
Also, why does this change?

Reply:
good call, I actually don't need to set that flag and all the tests pass, so removed it.